apps/admin_app/lib/admin_app_web/views/product_view.ex

```diff
@@ -29,7 +31,12 @@ defmodule AdminAppWeb.ProductView do
   end
 
   def has_themes(parent_product) do
-    length(parent_product.taxon.variation_themes) > 0
+    length(parent_product.taxon.variation_themes) > 0 && !check_if_child(parent_product)
```

Comment:
Remove call of `check_if_child(parent_product)` from `has_themes`. `has_themes` is intended to check if product has themes. Adding other function call to it will change it's intent.

Maybe we can add different function to check if the variation tab needs to be shown and call all function in that function.